index.js

```diff
@@ -5,6 +5,10 @@
 var restify = require('restify')
 var bufferize = require('./lib/bufferize')
 var version = require('./package.json').version
+var errors = require('./lib/error')
+
+var otherRoutes = /\/.+/
+var methods = [ 'get', 'put', 'post', 'del', 'head' ]
```

Comment:
If there's only one use of this list, it may be neater to do it inline like so:
```
['get', 'put', 'post', 'del', 'head'].forEach(function() {
...
})
```